bin/smoke-test.sh

```diff
@@ -528,6 +528,34 @@ requestMacaroon() { # $1 Caveats, $2 URL, $3 variable for macaroon, $4 variable
     eval $3="$macaroon"
 }
 
+requestWlcgToken() { # $1 oidc-agent account name, $2 audience, $3 variable for token, $4 token for result.
+    # Fallback to failing unless we specify otherwise.
+    eval $4=1
+    lastTestFailed=0
+
+    echo -n "Requesting token from account $1 for audience $2: "
+
+    token=$(oidc-token $1 --aud $2)
```

Comment:
I would suggest putting the aud argument in quotes; e.g.,

    token=$(oidc-token $1 --aud "$2")

This allows `$2` to be empty, which `oidc-agent` interprets as a request to use the OP's default audience claim.

Also, `$1` could be put in double-quote-marks, in case someone has an account shortname that has a space in it.